src/forms/CalendarComponent.jsx

```diff
@@ -114,20 +108,11 @@ CalendarComponent.propTypes = {
 	id: PropTypes.string,
 	name: PropTypes.string,
 	datepickerOptions: PropTypes.object,
-	error: PropTypes.oneOfType([
-		PropTypes.string,
-		PropTypes.element
-	]),
-	helperText: PropTypes.oneOfType([
-		PropTypes.string,
-		PropTypes.element
-	]),
+	error: PropTypes.oneOfType([PropTypes.string, PropTypes.element, PropTypes.bool]),
```

Comment:
only real change

Reply:
What's the use case for passing a boolean to `error`? I thought we used this prop for rendering an error message

Reply:
The schedule form is passing false when there is no error to show